### ISSUE TITLE
How to actually deploy this to Github Pages?

### ISSUE THREAD

**Issue by alvindizon** [mmacneil/devfolio#3]
How do I deploy this for my own Github Page, or in other words, see the actual devfolio site when accessing ```https://<your_git_username>.github.io```? It's not clear especially for those who are new to web development/those who just want to have a Github Pages up and running quickly. According to Github Pages you need to have ```index.html``` at your root directory, but for this template it's inside ```src/dist```, and trying to move ```index.html``` would mess things up. I managed to do [this hack](https://stackoverflow.com/a/25327759/4612653) but it involves a redirect.

**Comment by mmacneil** [comment]
Hey @alvindizon - here's the `gh-pages` branch as part of this repo that pages uses.  That will give you the structure so follow along with the rest of the docs to set up that branch on your repo. I can't remember the exact steps but I simply followed the docs so I think that approach should work for you.  Hope this helps.

![image](https://user-images.githubusercontent.com/381371/86918362-f7ce3780-c0fc-11ea-86d4-3640940df509.png)

**Comment by alvindizon** [comment]
So I just learned that there's a distinction between user pages and project pages. User pages are the ones that use your Github username(which is what I want), and can only publish from ```master```, while project pages use the ```gh-pages``` branch. What worked for me is to just create a separate branch for modifying the ```index.html``` and others, which I named ```dev```, and in the ```master``` branch, I just deleted everything and replaced them with the contents of the ```dist``` folder.
